src/Restivus/Extensions.cs

```diff
@@ -26,7 +30,11 @@ public static Uri FilterQueryParams(this Uri uri, params string[] keys)
             {
                 foreach (var key in keys)
                 {
+#if NETSTANDARD13
+                    if (queryParams.ContainsKey(key))
+#elif NET4_5
                     if (queryParams.AllKeys.Contains(key))
+#endif
```

Comment:
Yes, it would fail to compile.  The compile time constants are defined in the csproj files.